src/QMCWaveFunctions/Jastrow/TwoBodyJastrowOrbital.h

```diff
@@ -744,6 +688,7 @@ class TwoBodyJastrowOrbital: public OrbitalBase
 
   RealType ChiesaKEcorrection()
   {
+#if QMC_BUILD_LEVEL<5
```

Comment:
LEVEL=5 builds the executables in src/miniapps only. I think this was added for fair comparison between the old and new J2 implementation. In addition, based on Ray's claim, the Chiesa KE correction build on top of short range J2 is not theoretically correct and should be removed.